src/controllers/reportsController.js

```diff
@@ -1,4 +1,4 @@
-const reporthelper = require('../helpers/reportHelper')();
+const reporthelper = require('../helpers/reporthelper')();
```

Comment:
Critical change. Needs to be merged ASAP as it corrects the incorrect file name.